jobs/v3/howto/location_based_search_sample.go

```diff
@@ -142,9 +142,9 @@ func cityLocationSearch(w io.Writer, projectID, companyName, location string) (*
 	return resp, nil
 }
 
-// [END city_location_search]
+// [END job_city_location_search]
 
-// [START broadening_location_search]
+// [START job_broadening_location_search]
```

Comment:
you need to leave broadening_location_search here (both the start and end tags) until the new region tag has been merged in docs.

Reply:
Thank you for catching this. I revered this tag for now.